packages/swing-store/src/transcriptStore.js

```diff
@@ -31,22 +35,29 @@ function* empty() {
 }
 
 /**
- * @param {unknown} position
+ * @param {number} position
  * @returns {asserts position is number}
  */
 
 function insistTranscriptPosition(position) {
-  assert.typeof(position, 'number');
-  assert(position >= 0);
+  typeof position === 'number' || Fail`position must be a number`;
+  position >= 0 || Fail`position must not be negative`;
 }
 
 /**
  * @param {*} db
  * @param {() => void} ensureTxn
- * @param {(key: string, value: string) => void} noteExport
- * @returns { TranscriptStore & TranscriptStoreDebug }
+ * @param {(key: string, value: string | undefined ) => void} noteExport
+ * @param {object} [options]
+ * @param {boolean | undefined} [options.keepTranscripts]
+ * @returns { TranscriptStore & TranscriptStoreInternal & TranscriptStoreDebug }
  */
-export function makeTranscriptStore(db, ensureTxn, noteExport) {
+export function makeTranscriptStore(
+  db,
+  ensureTxn,
+  noteExport,
```

Comment:
nit:

provide a default like `makeSnapStore`?

```suggestion
  noteExport = () => {},
```